cli/cmd/cluster/apply.go

```diff
@@ -73,7 +89,10 @@ func Apply(contextLogger *log.Entry, options ApplyOptions) error {
 		ex:            c.terraformExecutor,
 	}
 
-	charts := platform.CommonControlPlaneCharts(options.UpgradeKubelets)
+	charts := c.platform.Meta().ControlplaneCharts
+	if !options.UpgradeKubelets {
+		charts = removeKubeletChart(charts)
```

Comment:
Very very soft suggestion, more like a thought, we could also write:
```suggestion
		charts = withoutKubeletChart(charts)
```

Or `withoutCharts(charts, []string{platform.KubeletChartName})`